templates/base.html

```diff
@@ -9,6 +9,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="author" content="Wakacyjne Warsztaty Wielodyscyplinarne">
 
+    <meta name="theme-color" content="#191919">
```

Comment:
Add to colors.md as WWW black?

Reply:
Actually... this is just `rgba(0, 0, 0, 0.9) * white background`, and it's a different color on desktop (0.8 instead of 0.9), which is why I didn't include it :joy: